404.html

```diff
@@ -2,7 +2,6 @@
 <html lang=en>
 	<head>
 		<title>404 - Page Not Found</title>
-		<base href="https://theforumhelpers.github.io"> <!--If making local changes, change the base tag to point to how your filesystem is set up-->
```

Comment:
if you remove the \<base> element, then things will break (#141)

for example, if you are on `theforumhelpers.github.io/foo/bar`, `resources/stylesheet.css` will direct to `theforumhelpers.github.io/foo/resources/stylesheet.css`. the \<base> element ensures relative urls point to the right place

Reply:
Sorry about that, resolving now.